### ISSUE TITLE
PDF reading order vs. visual order for Copy of Submission

### ISSUE THREAD

**Issue by BelovAndrew** [department-of-veterans-affairs/va.gov-team#118874]
### Description

Tag order already looks correct (screen reader use tag order). Nice-to-have: align visual and reading order in the export if feasible.

 The accordion section is somehow _first_ in the reading order, when it should come after the "Disability Compensation Claim" section (reading order should match visual order). The little numbers in the left corner of each content chunk is that items spot in the reading order:
  
  <img width="900" height="812" alt="Image" src="https://github.com/user-attachments/assets/9e4422bc-111e-4197-8b0f-86d4f78e161a" />

### How severe is the problem?

- [ ] Critical - A Veteran is blocked or significantly hindered from submitting the form, data is lost, or claim and evidence test coverage is missing. Fix ASAP.
- [ ] Moderate - A Veteran is not blocked, but the happy path is interrupted. Fix after critical bugs.
- [x] Minor - Cosmetic bugs and minor language inconsistencies. Defer to sprint planning.

### Definition of Done

- [ ] Meets acceptance criteria
- [ ] Passed all testing coverage
- [ ] Logging and monitoring in place
- [ ] Code or content reviewed (Internal)
- [ ] Reviewed and approved by product and/or design

### Refinement Checklist!

- [ ] Add description, tasks, and acceptance criteria
- [ ] Add estimate
- [ ] Add the proper labels, team, and priority
- [ ] Add an Epic or Super Epic
